.meteor/packages

```diff
@@ -69,6 +69,7 @@ kadira:flow-router-ssr
 matb33:collection-hooks
 meteorhacks:ssr
 meteorhacks:subs-manager
+natestrauser:select2
```

Comment:
@aaronjudd there's an NPM package for this (select2), but under its usage is a bunch of CDNs to source from, and alternative integrations, hence the use of the meteor package.